pipeline/bin/main.ml

```diff
@@ -12,7 +12,7 @@ module Source = struct
     in
     let github_token_path =
       Arg.required
-      @@ Arg.opt Arg.(some path) None
+      @@ Arg.opt Arg.(some file) None
```

Comment:
There's a slack path down a few lines which should also be a file, can you add that? Thanks! :)

Reply:
Done. I'm curious why isn't the slack path/URI passed as a string? Does it contain a secret?